mesh/config/cytoscape/cytoscape.go

```diff
@@ -40,6 +40,7 @@ type NodeData struct {
 	InfraData      interface{} `json:"infraData,omitempty"`      // infraType-dependent data
 	IsAmbient      bool        `json:"isAmbient,omitempty"`      // true if configured for ambient
 	IsBox          string      `json:"isBox,omitempty"`          // set for NodeTypeBox, current values: [ 'cluster', 'dataplanes', 'namespace' ]
+	IsCanary       bool        `json:"isCanary,omitempty"`       // true if the data plane is canary
```

Comment:
I guess another way we could have done this is to introduce another InfraType, like `MeshInfraType.DATAPLANECANARY`.  Although, I'm not sure that's better, this should be fine.

Reply:
I think that for the moment it is better to have IsCanary as an attribute. Currently both dataplanes are exactly the same except the badge, and adding new infratype would add more complexity in the frontend (add MeshInfraType.DATAPLANECANARY to all switches we have DATAPLANE). In a new mesh page iteration, if we decide to display different information for canary data plane, I am ok to add new infraType.

Reply:
No, I think 'isCanary` is fine, it was just a thought...